openhands/runtime/client/runtime.py

```diff
@@ -34,6 +35,62 @@
 from openhands.runtime.utils.runtime_build import build_runtime_image
 
 
+class LogBuffer:
+    """
+    Synchronous buffer for Docker container logs.
+
+    This class provides a thread-safe way to collect, store, and retrieve logs
+    from a Docker container. It uses a deque with a maximum length to store log
```

Comment:
- `deque` is now `list[]`
- remove `with a maximum length`

Reply:
fixed!